.github/workflows/main.yml

```diff
@@ -1,54 +1,65 @@
-name: 'CI Multiplatform Build'
+name: 'OBS (Push + Pull Request)'
```

Comment:
I'd personally vote just labelling this "Build". My current annoyance is quickly checking build status and not knowing which one is which, as the platform is documented after the unnecessarily long workflow name.

![image](https://user-images.githubusercontent.com/941350/115322518-9e398400-a1c9-11eb-80eb-6ff4c9950173.png)

Reply:
Right, will change that in an update later tonight.